client/__tests__/e2e/e2e.test.ts

```diff
@@ -449,7 +449,7 @@ describe("graph overlay", () => {
 });
 
 test("zoom limit is 12x", async ({ page }) => {
-  goToPage(page);
+  await goToPage(page);
```

Comment:
the fact I wasn't able to find this bug shows how test fatigued I was at the end of this 😆